app/models/report.rb

```diff
@@ -132,14 +135,10 @@ def no_report
     false
   end
 
-  def summaryStatus
-    return _("Failed")   if error?
-    return _("Modified") if changes?
-    _("Success")
-  end
-
-  # puppet report status table column name
   def self.report_status
     "status"
   end
 end
+
+require_dependency 'config_report'
```

Comment:
I must say that I cowardly copied from https://github.com/theforeman/foreman/blob/develop/app/models/template.rb#L68